### PR TITLE
feat(controller): add movie controller + test

### PR DESCRIPTION
**Pull request by lvamaral**
what: add a movie controller with the create route, and appropriate unit test for that controller
why: allows us to send post requests to add movies to db, and ensures controller is working through unit test

### CODE REVIEW

**Review by jesscxu — APPROVED** [review]
LGTM you just might need to rebase after merging your other PR

File: lib/models/movie.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Bookshelf = require('../libraries/bookshelf');
+const Bookshelf = require('../../libraries/bookshelf');
```
Comment (pallavi) [inline lib/models/movie.js]: were you able to run this locally with `yarn start`?

i'm getting an error that says this import can't be resolved

**Comment by lvamaral** [comment]
@pallavi there were some inconsistencies with other PRs in the location of that file, but I think I fixed it for this one!

**Review by pallavi — APPROVED** [review]
ah ok as long as it works when you merge into master :) LGTM!

**Pull request merged**